.spelling

```diff
@@ -389,6 +389,8 @@ secretName
 selinux
 serviceaccount
 servicegraph-example
+servicerole
```

Comment:
Do not add these words to the . spelling
Please write the words separately in the text: service role and service role binding.
If you are making a direct reference to the value of the `kind:` key, then they should be surrounded by back-tics and capitalized accordingly. Thus, they do not need to be added.

Reply:
This is needed for the tag reference in line 775.

Reply:
That is just my point, it should not be needed. Please change the content on the file you mention for the entries here not to be needed.

Reply:
Removed.